### PR TITLE
setup and configured sequlize

### PR DESCRIPTION
**Pull request by Anasauce**
@deonna setup configuration for sequlize

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
     "pg": "^6.1.0",
     "pg-promise": "^5.3.5",
     "pug": "^2.0.0-beta6",
-    "serve-favicon": "~2.3.0"
+    "sequelize": "^3.27.0",
+    "sequelize-cli": "^2.4.0",
+    "serve-favicon": "~2.3.0",
+    "sqlite3": "^3.1.8"
```
Comment (deonna) [inline package.json]: Any reason why you're adding `sqlite3` here instead of just sticking with `PostgreSQL`?

File: config/config.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "root",
+    "password": null,
+    "database": "database_development",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
```
Comment (Anasauce) [inline config/config.json]: we just changed the dialect to "postgres" in the config/config.json file here...do you think there is any other place where we need to make changes in the configuration?

File: config/config.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "database": "database_test",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
+  },
+  "production": {
+    "username": "root",
+    "password": null,
+    "database": "database_production",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
```
Comment (deonna) [inline config/config.json]: Postgres.

File: config/config.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "database": "database_development",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
+  },
+  "test": {
+    "username": "root",
+    "password": null,
+    "database": "database_test",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
```
Comment (deonna) [inline config/config.json]: Postgres.

File: package.json (hunk 1 above)
Comment (deonna) [inline package.json]: Probably don't need this since you're using Postgres.

File: config/config.json (hunk 2 above)
Comment (deonna) [inline config/config.json]: I replied to all the lines that need to be changed or deleted - let me know if you have any questions!

**Pull request merged**